package.json

```diff
@@ -32,7 +32,7 @@
     "test:specs": "tsx scripts/specs.ts",
     "test:types": "npm run --workspaces test:types && tsc",
     "test": "npm run test:caniuse -- --quiet && npm run test:schema && npm run test:specs && npm run test:format && npm run test:dist && npm run test --workspaces && npm run test:lint",
-    "update-drafts": "tsx scripts/update-drafts.ts"
+    "update-drafts": "tsx scripts/update-drafts.ts && npm run dist"
```

Comment:
I see the benefit here, but this is going to make it hard to pass arguments to `update-drafts`. Could we keep `update-drafts` only invoking that script, and then add `update-drafts-dist` that does both?

Alternately, it isn't terribly hard to run `npx tsx scripts/update-drafts.ts --keys color`.

Reply:
I didn't realize anyone was doing this locally as a routine matter. I've undone that bit with 5c8ea95d. 👍